test/ut/ut.cpp

```diff
@@ -717,6 +735,16 @@ int main() {
       reporter = printer{};
     }
 
+    {
+      std::size_t summary_count = 0;
```

Comment:
I ran this test case, before applying the fix, it appropriately fails and identified the error. I applied the fix, and the test then passes. :-)